lib/inferno/apps/cli/execute.rb

```diff
@@ -118,6 +134,33 @@ def run_one(runnable, test_run)
         dispatch_job(test_run)
       end
 
+      def inputs_and_preset
+        if preset
+          preset_inputs = preset.inputs.to_h do |preset_input|
+            [preset_input[:name], preset_input[:value]]
+          end
+
+          options.fetch(:inputs, {}).merge(preset_inputs)
```

Comment:
I think this should go the other way, so that any explicitly defined inputs override the preset.